Library/Homebrew/cmd/audit.rb

```diff
@@ -360,12 +360,8 @@ def audit_desc
       EOS
     end
 
-    if desc =~ %r[[Cc]ommandline]
-      problem "It should be \"command-line\", not \"commandline\"."
-    end
-
-    if desc =~ %r[[Cc]ommand line]
-      problem "It should be \"command-line\", not \"command line\"."
+    if desc =~ %r[([Cc]ommand ?line)]
+      problem "It should be \"command-line\", not \"#{$1}\""
```

Comment:
I know it's the existing logic to be blamed. But maybe `"It should be \"command-line\", not \"#{$1}\" in description"` is better, so we know such problem comes from description.